tools/postinstall/apply-patches.js

```diff
@@ -14,7 +14,7 @@ const chalk = require('chalk');
  * Version of the post install patch. Needs to be incremented when
  * existing patches or edits have been modified.
  */
-const PATCH_VERSION = 6;
+const PATCH_VERSION = 9;
```

Comment:
And now we are going to `9` due to the revert. For later reference: 7 and 8 were polluted due to the initial PR. Now we changed the patch again, so we are at 9.

Reply:
👍